### PR TITLE
Table improvements

### PR DESCRIPTION
**Pull request by aredridel**

### CODE REVIEW

**Comment by aredridel** [comment]
screen shot 2016-05-09 at 17 19 47

**Comment by aredridel** [comment]
@stubbornella @jefflembeck review?

File: src/pivotal-ui/components/tables/tables.scss [hunk 1]
```diff
@@ -64,6 +64,10 @@ table, .table {
   @include table-horz-divided();
 }
 
+.table > tbody + tbody {
```
Comment (jefflembeck) [inline src/pivotal-ui/components/tables/tables.scss:67]: I'm mostly just curious at this point, but in what situation do we run into:

```
<table class="table">
    <tbody></tbody>
    <tbody></tbody>
</table>
```

?

Comment (aredridel) [inline src/pivotal-ui/components/tables/tables.scss:67]: Not very many, but it's potentially a thing.

Comment (jefflembeck) [inline src/pivotal-ui/components/tables/tables.scss:67]: Ooof. I hate that potential thing.

**Comment by jefflembeck** [comment]
This sure is nice lookin.

**Comment by aredridel** [comment]
Thanks!

**Comment by aredridel** [comment]
No divisions, white background:

screen shot 2016-05-10 at 16 32 09

Divisions, default:
screen shot 2016-05-10 at 16 31 22

No divisions, default background screen shot 2016-05-10 at 16 31 14

File: src/pivotal-ui/components/tables/tables.scss (hunk 1 above)
Comment (stubbornella) [inline src/pivotal-ui/components/tables/tables.scss:67]: hmm... I wonder if we could get away with not covering this eventuality?

**Comment by stubbornella** [comment]
Can't wait to use them! 🐑

**Pull request merged**